src/components/components.scss

```diff
@@ -12,4 +12,5 @@
   'Navbar/Navbar',
   'Sidebar/Sidebar',
   'TextArea/TextArea',
+  'Builder/Breadcrumbs/Breadcrumbs',
```

Comment:
Lets not put this component in the Builder directory, lets just have it live in the components